public/session-storage.js

```diff
@@ -0,0 +1,83 @@
+(function collectAnswers () {
+
+  // select survey type
+  if (location.pathname === '/select-survey') {
+    var preSession = document.getElementById('pre-session-button')
+    preSession.addEventListener('click', function () {
+      sessionStorage.setItem('survey', 'pre-session')
+    })
+
+    var postSession = document.getElementById('post-session-button')
+    postSession.addEventListener('click', function () {
+      sessionStorage.setItem('survey', 'post-session')
+    })
+  }
+
+  // obtain consent
+  if (location.pathname === '/consent') {
+    var yes = document.getElementById('yes')
```

Comment:
There is only one "yes" option in partials

It gets reused in different contexts (hence the if statements), so I can't make the id specific like "agree-consent"

Should it be a class then?